src/ai/ai.controller.ts

```
@@ -0,0 +1,80 @@
+import { Body, Controller, Param, Post, Query } from '@nestjs/common';
+import { AiService } from './ai.service';
+import { GameService } from '../modules/game/game.service';
+
+class SuggestBodyDto {
+  fen?: string;   // optional: override server FEN (analysis mode); otherwise read from game
```

ESLint

[error] Delete `··`

(prettier/prettier)